purchase_order_line_schedule_manual_delivery/models/purchase_order_line_schedule.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+from odoo.tools import float_compare
+
+from odoo.addons import decimal_precision as dp
+
+
+class PurchaseOrderLineSchedule(models.Model):
+    _inherit = "purchase.order.line.schedule"
+
+    existing_qty = fields.Float(
+        compute="_compute_existing_qty",
+        string="Receiving",
+        compute_sudo=True,
+        digits=dp.get_precision("Product Unit of Measure"),
+        help="Quantity already planned or shipped " "(stock movements already created)",
+    )
+    pending_to_receive = fields.Boolean(
+        compute="_compute_existing_qty",
+        compute_sudo=True,
+        string="Pending to Receive",
+        help="There is pending quantity to receive not yet planned",
+    )
+
+    def _compute_existing_qty(self):
```

Comment:
Is this not double, same method available in purchase_order_line_schedule_stock and this module depends on purchase_order_line_schedule_stock?